pkg/handlers/login.go

```diff
@@ -385,6 +385,17 @@ func (h *Handler) OIDCLoginCallback(w http.ResponseWriter, r *http.Request) {
 	}
 	http.SetCookie(w, &tokenCookie)
 
+	signedTokenCookie, err := session.GetSessionCookie(responseToken, expire, "")
+	if err != nil {
+		logger.Error(errors.Wrap(err, "failed to get session cookie"))
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	if u.Scheme == "https" {
+		signedTokenCookie.Secure = true
+	}
+	http.SetCookie(w, signedTokenCookie)
```

Comment:
what's the reason for passing an empty string to the function? also, doesn't the function handle whether or not `Secure` is set for the cookie?

Reply:
could maybe instead pass `u.String()` and let the function handle it?

Reply:
The third parameter for that function is the origin url and that's how it determines if it's secure or not. I left it empty and check the `u.Scheme` instead so that it was consistent with the others in this function but I can pass in the `redirectURL` and it would be the same result too.